src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2712,7 +2712,7 @@ private static bool ContainsModifier(SyntaxTokenList modifiers, SyntaxKind modif
             }
             else
             {
-                membersByName = membersAndInitializers.NonTypeMembers.ToDictionary(s => s.Name, StringOrdinalComparer.Instance);
```

Comment:
>StringOrdinalComparer.Instance [](http://example.com/codeflow?start=96&length=30)

Please confirm that this comparer and the default comparer are equivalent.

Reply:
they're the same.  StringOrdinalComparer.Instance override Equals as `return string.Equals(a, b);` which will be same Equals as the default.